src/api/searchApi.ts

```diff
@@ -224,40 +229,53 @@ export const fetchEmployees = async (
   return getEmployees.data;
 };
 
-export const fetchNviCandidates = async (results: number, from: number, query = '') => {
-  const paginationQuery = `size=${results}&offset=${from}`;
-  const fullQuery = [query, paginationQuery].filter(Boolean).join('&');
-
-  const getNviCandidates = await authenticatedApiRequest2<NviCandidateSearchResponse>({
-    url: `${SearchApiPath.NviCandidate}?${fullQuery}`,
-  });
-
-  return getNviCandidates.data;
-};
-
 enum NviCandidatesSearchParam {
+  Affiliations = 'affiliations',
   Aggregation = 'aggregation',
+  Assignee = 'assignee',
+  ExcludeSubUnits = 'excludeSubUnits',
+  Filter = 'filter',
   Offset = 'offset',
+  Query = 'query',
   Size = 'size',
   Year = 'year',
 }
 
 export interface FetchNviCandidatesParams {
-  [NviCandidatesSearchParam.Aggregation]?: 'all' | 'organizationApprovalStatuses' | null;
+  [NviCandidatesSearchParam.Affiliations]?: string[] | null;
+  [NviCandidatesSearchParam.Aggregation]?: 'all' | keyof NviCandidateAggregations | null;
+  [NviCandidatesSearchParam.Assignee]?: string | null;
+  [NviCandidatesSearchParam.ExcludeSubUnits]?: boolean | null;
+  [NviCandidatesSearchParam.Filter]?: keyof NviCandidateAggregations | null;
   [NviCandidatesSearchParam.Offset]?: number | null;
+  [NviCandidatesSearchParam.Query]?: string | null;
   [NviCandidatesSearchParam.Size]?: number | null;
   [NviCandidatesSearchParam.Year]?: number | null;
 }
```

Comment:
Jeg syns det er veldig ryddig å ha et slikt objekt, fordi det viser oss hvilke valg vi har når vi skal gjøre kallet (uten at vi trenger å gå inn i swagger for å sjekke)